MonoGame.Framework/Input/GamePad.UWP.cs

```diff
@@ -9,6 +9,12 @@ namespace Microsoft.Xna.Framework.Input
 {
     static partial class GamePad
     {
+        /// <summary>
```

Comment:
It's fine if you want to leave a comment, but this is a private variable, there shouldn't be a summary, no one is ever gonna read a summary for a private variable...